satpy/readers/clavrx.py

```diff
@@ -92,29 +93,29 @@ def _get_rows_per_scan(sensor: str) -> Optional[int]:
     return None
 
 
-def _remove_attributes(attrs: dict) -> dict:
-    """Remove attributes that described data before scaling."""
-    old_attrs = ['unscaled_missing', 'SCALED_MIN', 'SCALED_MAX',
-                 'SCALED_MISSING']
-
-    for attr_key in old_attrs:
-        attrs.pop(attr_key, None)
-    return attrs
-
-
 class _CLAVRxHelper:
     """A base class for the CLAVRx File Handlers."""
 
     @staticmethod
-    def _scale_data(data_arr: xr.DataArray, scale_factor: float, add_offset: float) -> xr.DataArray:
+    def _remove_attributes(attrs: dict) -> dict:
+        """Remove attributes that described data before scaling."""
+        old_attrs = ['unscaled_missing', 'SCALED_MIN', 'SCALED_MAX',
+                     'SCALED_MISSING']
+
+        for attr_key in old_attrs:
+            attrs.pop(attr_key, None)
+        return attrs
+
+    @staticmethod
+    def _scale_data(data_arr, scale_factor: float, add_offset: float) -> xr.DataArray:
```

Comment:
Is the removal of the type hint on `data_arr` intentional?

Reply:
If the valid_range isn't a DataArray then I guess `Union[xr.DataArray, list, tuple]`. Or you could always force `valid_range` to a tuple when calling this method and then do `Union[xr.DataArray, tuple[float, float]]` (or remove the sub-types on the tuple).